### ISSUE TITLE
Line numbers get out of sync after a long string variable

### ISSUE THREAD

**Issue by tremby** [sasstools/sass-lint#1075]
I have the following at the top of a scss file:

```scss
//add items to this list if they should be styled as text inputs or textareas.
$formTextSelectors: 'input[type="text"],
  input[type="email"],
  input[type="password"],
  input[type="date"],
  input[type="file"],
  input[type="number"],
  input[type="url"],
  input[type="tel"],
  select,
  textarea,
  .react-phone-number-input__phone';

form {
  $root: &;

  label {
    display: block;
    // ...
  }
  // ...
}
```

I'm getting some warnings I can't understand from this:

```
src/scss/component/_form.scss|10 col 32| Space expected around operator [Warning/space-around-operator]
src/scss/component/_form.scss|14 col 33| Space expected around operator [Warning/space-around-operator]
src/scss/component/_form.scss|15 col 35| Space expected around operator [Warning/space-around-operator]
```

Line 10 is the `select` line of the long string. Line 14 is the `form {` line. Line 15 is `$root: &;`. I don't know why this rule would give a positive on any of those lines.

**Comment by tremby** [comment]
Looking closer, they do point to actual problems in the file (after the listed code) if I collapse the long string to a single line. So it seems like line numbers are getting out of sync if there is a multi-line string.

**Comment by DanPurdy** [comment]
Possible AST issue will look to confirm.

**Comment by bgriffith** [comment]
This is fixed in the current version `v1.11.0`.
